packages/kbn-i18n/src/loader.ts

```diff
@@ -123,19 +120,19 @@ export async function getTranslationsByLocale(locale: string): Promise<Translati
   }
 
   if (!files.length) {
-    return { messages: {} };
+    return { locale, messages: {} };
   }
 
   return files.reduce(
-    (translation: Translation, file) => ({
+    (translation: TranslationInput, file) => ({
```

Comment:
It's weird that TranslationInput.messages is defined upstream as
```typescript
    messages: Record<MessageIds, string> | Record<MessageIds, MessageFormatElement[]>;
```

So it's either one or the other, but not mixed?
If we always have the same type on our side we might want to better delimit `messages` type.

Note that TS is currently failing here.

Also, I wanted to suggest using `files.reduce<TranslationInput>(...)` instead of the `as Translation Input`